ingestion-beam/src/test/java/com/mozilla/telemetry/SinkMainTest.java

```diff
@@ -103,28 +103,6 @@ public void canWriteSingleRecord() {
     assertThat(outputLines, matchesInAnyOrder(inputLines));
   }
 
-  @Test
-  public void canWriteErrorOutput() {
```

Comment:
We remove this test entirely because it was only testing handling of malformed PubsubMessage-serialized-to-json messages, a format that doesn't occur in production.